Backend/DataStore.php

```diff
@@ -9,6 +9,7 @@ class DataStore implements DataStoreInterface
 
     const CLIENT_API_KEY = "clientApiKey";
     const CLOUDFLARE_EMAIL = "cloudflareEmail";
+    const ZONE_ID_KEY = "zoneId:";
```

Comment:
Yeah I want to be able to store multiple zoneIds in the form zoneId:[domainName].